tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "@sindresorhus/tsconfig",
+	"compilerOptions": {
+		"esModuleInterop": true,
+		"noImplicitAny": true,
+		"declaration": true,
```

Comment:
This is already defined in https://github.com/sindresorhus/tsconfig/blob/master/tsconfig.json